### PR TITLE
Add offload option in flux-control training

### PR DESCRIPTION
**Pull request by Adenialzz**
Set model offloading to be optional in args, which will reduce the training memory usage but also reduce the training speed.

@sayakpaul

### CODE REVIEW

File: examples/flux-control/train_control_flux.py [hunk 1]
```diff
@@ -541,6 +541,7 @@ def parse_args(input_args=None):
         default=1.29,
         help="Scale of mode weighting scheme. Only effective when using the `'mode'` as the `weighting_scheme`.",
     )
+    parser.add_argument("--offload", action="store_true", help="Whether to do model offloading")
```
Comment (sayakpaul) [inline examples/flux-control/train_control_flux.py:544]:
```suggestion
    parser.add_argument("--offload", action="store_true", help="Whether to offload the VAE and the text encoders to CPU when they are not used.")
```

**Review by sayakpaul — APPROVED** [review]
Nice, thank you!

**Comment by sayakpaul** [comment]
Could you run `make style && make quality`?

**Comment by Adenialzz** [comment]
fixed now. ruff version issue

**Comment by sayakpaul** [comment]
Thanks for your PR!

**Pull request merged**